airgun/views/computeprofile.py

```diff
@@ -0,0 +1,26 @@
+from widgetastic.widget import Text, TextInput
+from airgun.views.common import BaseLoggedInView, SearchableViewMixin
+from airgun.widgets import ActionsDropdown
+
+
+class ComputeProfileView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='Compute Profiles']")
+    new = Text("//a[contains(@href, '/compute_profiles/new')]")
+    rename = Text("//a[contains(@href, '/compute_profiles/') and "
+                  "contains(@href, 'edit')]")
+    actions = ActionsDropdown("//td//div[contains(@class, 'btn-group')]")
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class ComputeProfileDetailsView(BaseLoggedInView):
```

Comment:
`ComputeProfileCreateView` is more appropriate name, as edit functionality has completely another appearance.
BTW, maybe you like to implement `edit` view as well, but it can be a tough task

Reply:
however, it is probably, not edit actually, but like actually operating and applying correct parameters. Anyway, I am talking about that view:
![image](https://user-images.githubusercontent.com/10895065/39427720-860d922a-4c8d-11e8-8fd6-a06f3c640d84.png)

Reply:
I think It should be easier to implement this after the compute resources will be finally added. 
If I am understand properly,  you should implement test which create some resource and than apply the profile on some resource and change the parameters.